new_images.yml

```diff
@@ -1,7 +1,7 @@
 ---
 new_images:
-  - spark: "3.2"
+  - spark: "3.3"
     use-case: "processing"
     processors: ["cpu"]
     python: ["py39"]
-    sm_version: "1.1"
+    sm_version: "1.0"
```

Comment:
Why are we going down to 1.0?

Reply:
As discussed offline, this represents container's minor version. Hence, we downgrade it to 1.0 everytime major version gets upgraded because of spark or python version update.